### PR TITLE
Clean up Kapacitor Alert configuration for Telegram

### PR DESCRIPTION
**Pull request by lukevmorris**
- [x] CHANGELOG.md updated
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #921

## The problem
Telegram configuration has proved to be confusing for users.

## The Solution
See [this list](https://github.com/influxdata/chronograf/issues/921#issuecomment-290509249) for all the changes.

### QuestionMarkTooltip
The most notable change is the addition of the [QuestionMarkTooltip component](https://github.com/influxdata/chronograf/pull/1135/files#diff-2d47a77591e6830c64d476c9a4a92910). Places an inline `?` on the page, which reveals a tooltip on hover. The tooltip remains in view while you're hovering over the `?`, or on the tooltip itself. This allows us to mouse over the tooltip and click any links contained within the tooltip:
hoverable-tooltips

### CODE REVIEW

File: ui/src/kapacitor/components/TelegramConfig.js [hunk 1]
```diff
@@ -1,18 +1,24 @@
 import React, {PropTypes} from 'react'
+import QuestionMarkTooltip from 'src/shared/components/QuestionMarkTooltip'
+const {
+  bool,
+  func,
+  shape,
+  string,
+} = PropTypes
```
Comment (121watts) [inline ui/src/kapacitor/components/TelegramConfig.js]: 👍  be the change

File: ui/src/kapacitor/components/TelegramConfig.js [hunk 2]
```diff
@@ -50,49 +55,76 @@ const TelegramConfig = React.createClass({
...
           </div>
 
           <div className="form-group col-xs-12">
-            <label htmlFor="chat-id">Chat ID</label>
-            <input className="form-control" id="chat-id" type="text" ref={(r) => this.chatID = r} defaultValue={chatID || ''}></input>
+            <label htmlFor="chat-id">
+              Chat ID
+              <QuestionMarkTooltip
+                tipID="chat-id"
+                tipContent="<p>Need help finding your chat id? Check out <a href='https://docs.influxdata.com/kapacitor/latest/guides/event-handler-setup/#telegram-chat-id' target='_blank'>these steps</a>.</p>"
```
Comment (121watts) [inline ui/src/kapacitor/components/TelegramConfig.js]: What do you think about pulling this text into a `/copy` directory and importing them?  Or, some other strategy that doesn't include this long of a string in a prop?

File: ui/src/shared/components/QuestionMarkTooltip.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+const {
+  string,
+} = PropTypes
+
+QuestionMarkTooltip.propTypes = {
+  tipID: string.isRequired,
+  tipContent: string.isRequired,
+}
+
+export default QuestionMarkTooltip
```
Comment (121watts) [inline ui/src/shared/components/QuestionMarkTooltip.js]: Love this.  Looking forward to using it.

File: ui/src/kapacitor/components/TelegramConfig.js (hunk 2 above)
Comment (cryptoquick) [inline ui/src/kapacitor/components/TelegramConfig.js]: I like that idea. importing copy. Could be a good route towards i18n?

Comment (lukevmorris) [inline ui/src/kapacitor/components/TelegramConfig.js]: Elementary, my dear @121watts -- I made these constants and moved them to `src/kapacitor/copy.js`

**Pull request merged**